types/node/v12/worker_threads.d.ts

```diff
@@ -60,6 +60,7 @@ declare module "worker_threads" {
         stdout?: boolean;
         stderr?: boolean;
         execArgv?: string[];
+        env?: symbol;
```

Comment:
I think #41356 has a better solution in this as besides `worker.SHARE_ENV` also an object is possible here.